packages/access-test-model/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# Based on spack/var/spack/repos/builtin/packages/fiat/package.py
+
+from spack.package import *
+
+
+class AccessTestModel(CMakePackage):
+    """ACCESS Test Model is a small test FORTRAN model component
+       used to test CI workflows."""
+
+    homepage = "https://github.com/ACCESS-NRI/ACCESS-TEST-model"
+    git = "https://github.com/ACCESS-NRI/ACCESS-TEST-model.git"
+
+    maintainers("aidanheerdegen", "codegat", "harshula")
+
+    license("Apache-2.0")
+
+    version("main", branch="main", no_cache=True)
```

Comment:
Hi @CodeGat , This might be a workaround for our stale git repo issue when using branches.

Reply:
I'll resolve this so I can merge without bypassing checks.